google/cloud/firestore_v1/_helpers.py

```diff
@@ -70,6 +70,20 @@
 }
 
 
+class EmptyParam:
```

Comment:
This is a little more trouble than I usually go to for this. Generally something like:

_not_passed = object()

def f(x, y=_not_passed):
    if y is _not_passed:
        etc...

Reply:
Oh yeah, that's less lines :)